docs/source/using/experimental-features.adoc

```diff
@@ -151,3 +151,52 @@ To get the status of the DNS server:
 ----
 $ minishift dns status
 ----
+
+[[run-against-an-existing-machine]]
+== Run against an Existing Machine
+
+{project} can be run against an existing machine using `vm-driver` as `generic` along with the required options.
+To use an existing machine with {Project} , it need to be configured as follows:
+
+[NOTE]
+====
+CentOS/RHEL/Fedora are the suggested Linux distributions for this feature.
```

Comment:
@praveenkumar I think we would be better off saying s/CentOS/CentOS 7/ , similarly Fedora (> 26) and RHEL7. WDYT?